enterprise/server/cmd/ci_runner/main.go

```diff
@@ -2131,6 +2206,21 @@ func gitRemoteName(repoURL string) string {
 	return forkGitRemoteName
 }
 
+func runBashCommand(ctx context.Context, cmd string, env map[string]string, dir string, outputSink io.Writer) (string, *commandErr) {
+	if err := printCommandLine(outputSink, cmd); err != nil {
+		return "", &commandErr{err, ""}
+	}
+
+	var buf bytes.Buffer
+	w := io.MultiWriter(outputSink, &buf)
+
+	if err := runCommand(ctx, "bash", []string{"-c", cmd}, env, dir, w); err != nil {
```

Comment:
add `-eo pipefail` since it's a better default and also more familiar:

```suggestion
	if err := runCommand(ctx, "bash", []string{"-eo", "pipefail", "-c", cmd}, env, dir, w); err != nil {
```